pkg/exchange/okex/query_kline_test.go

```diff
@@ -0,0 +1,74 @@
+package okex
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryKlines(t *testing.T) {
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+	}
+
+	e := New(key, secret, passphrase)
+
+	queryOrder := types.OrderQuery{
+		Symbol: "BTC-USDT",
+	}
+
+	now := time.Now()
+	// test supported interval - minute
+	klineDetail, err := e.QueryKLines(context.Background(), queryOrder.Symbol, types.Interval("1m"), types.KLineQueryOptions{
+		Limit:   50,
+		EndTime: &now})
+	if assert.NoError(t, err) {
+		assert.NotEmpty(t, klineDetail)
+	}
+	t.Logf("kline detail: %+v", klineDetail)
```

Comment:
rm, the log will increase the storage size

Reply:
ok, but I think we didn't do any unit test while deploying ?

Reply:
The unit test is necessary.
The log messages are not necessary in unit test. Unless you want to debug.

Reply:
Oh, I understand why you are said ` but I think we didn't do any unit test while deploying `. If you retrieve data from the actual server, it is considered an integration test, not a unit test.

A unit test typically involves creating a request and invoking the mock server, then confirming that the response matches the expected outcome.